hbase-client/src/main/java/org/apache/hadoop/hbase/MetaTableAccessor.java

```diff
@@ -1819,7 +1819,8 @@ private static void updateLocation(Connection connection, RegionInfo regionInfo,
    * @param regionInfo region to be deleted from META
    * @throws IOException
    */
-  public static void deleteRegion(Connection connection, RegionInfo regionInfo) throws IOException {
+  public static void deleteRegionInfo(Connection connection, RegionInfo regionInfo)
```

Comment:
Why change the method name?

Reply:
Because  the method only delete the info column family (will keep other cf), so name it deleteRegionInfo will be better.